Tests/StreamChatTests/Dummy data/TestAttachmentEnvelope.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+
+struct TestAttachmentEnvelope: AttachmentEnvelope, Equatable, Decodable {
+    var type: AttachmentType = .custom("test")
+    let name: String
+    let number: Int
+    
+    init(name: String = "Test", number: Int = .random(in: 1...100)) {
```

Comment:
nit: `name: String = .unique`

Reply:
Why do we use `init` instead of `static func dummy()` like everywhere else? :)

Reply:
We usually using `.dummy()` for types introduced in SDK and here I decided that the whole type is `dummy` so `init` is ok :)

Reply:
ok 🤷‍♂️ :)